rocket-chat-android-widgets/src/main/res/layout/message_inline_attachment.xml

```diff
@@ -1,41 +1,109 @@
+                android:id="@+id/author_box"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:gravity="center_vertical"
+                android:orientation="horizontal"
+                android:layout_marginBottom="8dp">
+
+            <ImageView
+                    android:id="@+id/author_icon"
+                    android:layout_width="16dp"
+                    android:layout_height="16dp"
+                    tools:src="@drawable/circle_black" />
+
+            <android.support.v4.widget.Space
+                    android:layout_width="8dp"
+                    android:layout_height="8dp" />
+
+            <TextView
+                    android:id="@+id/author_name"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:textAppearance="@style/TextAppearance.RocketChat.MessageAttachment.Title.Link"
+                    tools:text="Bradley Hilton" />
+
+            <android.support.v4.widget.Space
+                    android:layout_width="8dp"
+                    android:layout_height="8dp" />
+
+            <TextView
+                    android:id="@+id/timestamp"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    tools:text="14:53" />
+
+        </LinearLayout>
+
         <TextView
                 android:id="@+id/title"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_marginTop="2dp"
-                android:layout_marginBottom="2dp"
+                android:layout_marginBottom="8dp"
                 android:textAppearance="@style/TextAppearance.RocketChat.MessageAttachment.Title"
-                android:background="?attr/selectableItemBackground" />
+                android:background="?attr/selectableItemBackground"
+                tools:text="Attachment Example" />
+
+        <LinearLayout
+                android:id="@+id/ref_box"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:gravity="center_vertical"
+                android:orientation="horizontal"
+                android:layout_marginBottom="8dp">
+
+            <ImageView
+                    android:id="@+id/thumb"
+                    android:layout_width="32dp"
+                    android:layout_height="32dp"
+                    tools:src="@drawable/circle_black" />
+
+            <android.support.v4.widget.Space
+                    android:layout_width="8dp"
+                    android:layout_height="8dp" />
+
+            <TextView
+                    android:id="@+id/text"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    tools:text="Bradley Hilton" />
```

Comment:
That's the same name used on the attachment documentation. I think it's quite OK. xD